setup.py

```diff
@@ -40,13 +40,13 @@
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.4",
```

Comment:
You'll need to update python_requires if we want to actually drop support for 3.4

Reply:
Done.  PyMongo 3.13 doesn't support 3.4, which is why we have to drop it.

Reply:
I think you forgot to push.

Reply:
Correct, done